.github/workflows/build-node-packages.yml

```diff
@@ -0,0 +1,202 @@
+      run: |
+        Write-Host "Fake step that do nothing"
+        Write-Host "We need it because log of previous step 'Setup Node' is not available here yet."
+        Write-Host "In testing step (Node.Tests.ps1) we analyze build log of 'Setup Node' task"
+        Write-Host "to determine if Node.js version was consumed from cache and was downloaded"
+      shell: pwsh
+
+    - name: Run tests
+      run: |
+        Install-Module Pester -Force -Scope CurrentUser -RequiredVersion 4.10.1
+        Import-Module Pester
+        $pesterParams = @{
+          Path="./Node.Tests.ps1";
+          Parameters=@{
+            Version="$env:VERSION";
+          }
+        }
+        Invoke-Pester -Script $pesterParams -EnableExit -OutputFile "test_results.xml" -OutputFormat NUnitXml
+      working-directory: ./tests
+      shell: pwsh
+
+  publish_release:
+    name: Publish release
+    if: github.event.inputs.PUBLISH_RELEASES == 'true'
+    needs: test_node
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+    - name: Publish Release ${{ env.VERSION }}
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ env.VERSION }}-${{ github.run_id }}
+        release_name: ${{ env.VERSION }}
+        body: |
+          Upload Node.js ${{ env.VERSION }}
+
+  upload_assets:
+    name: Upload assets for ${{ matrix.platform }}
+    needs: publish_release
+    runs-on: ubuntu-latest
+    env:
+      UPLOAD_URL: ${{ needs.publish_release.outputs.upload_url }}
+    strategy:
+      matrix:
+        extension: ['tar.gz']
+        platform: [linux, darwin] 
+        include:
+        - platform: win32
+          extension: 7z
+    steps:
+    - uses: actions/download-artifact@v2
+      with:
+        name: node-${{ env.VERSION }}-${{ matrix.platform }}
+
+    - name: Upload Release Asset
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ env.UPLOAD_URL }}
+        asset_path: ./node-${{ env.VERSION }}-${{ matrix.platform }}-${{ env.ARCHITECTURE }}.${{ matrix.extension }}
+        asset_name: node-${{ env.VERSION }}-${{ matrix.platform }}-${{ env.ARCHITECTURE }}.${{ matrix.extension }}
+        asset_content_type: application/zip
+
+  create_pr:
+    name: Create Pull Request
+    needs: upload_assets
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Trigger "Create Pull Request" workflow
+      uses: actions/github-script@v2
+      with:
+        github-token: ${{ secrets.PERSONAL_TOKEN }}
+        script: |
+          # TODO: currently 'actions.createWorkflowDispatch' function does not work. I will investigate  
+          github.repos.createDispatchEvent({
+            owner: context.repo.owner,
```

Comment:
`context.repo.owner` and `context.repo.repo` are available only inside github-script?